tutorials/Jak korzystac z GitHub.md

```diff
@@ -0,0 +1,193 @@
+# Dokument ma na celu wytłumaczenie w jaki sposób korzystać z GitHuba.
```

Comment:
To nazwa sekcji - a nie brzmi jak tytuł sekcji. Obecnie mamy jedną sekcje zawierającą subsekcje. Może warto poprawić podział by cały dokument nie był po prostu jedną sekcją.

Reply:
Nie wiem, czy o to chodziło?

Reply:
Trochę tak, a trochę nie. Teraz cały dokument składa się tylko z subsekcji (podwójny hash) a nie ma sekcji w ogóle (pojedynczy hash)